c7n/resources/sagemaker.py

```diff
@@ -140,6 +140,32 @@ def _augment(j):
         return list(map(_augment, super(SagemakerTransformJob, self).augment(jobs)))
 
 
+class DescribeJob(DescribeSource):
+
+    def augment(self, resources):
+        return universal_augment(self.manager, super().augment(resources))
+
+
+@resources.register('sagemaker-labeling-job')
+class SagemakerLabelingJob(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        arn_type = "labeling-job"
+        service = 'sagemaker'
+        enum_spec = ('list_labeling_jobs', 'LabelingJobSummaryList', None)
+        detail_spec = (
+            'describe_labeling_job', 'LabelingJobName', 'LabelingJobName', None)
+        arn = id = 'LabelingJobArn'
+        name = 'LabelingJobName'
+        date = 'CreationTime'
+        universal_taggable = object()
+        permission_augment = ('sagemaker:DescribeLabelingJob', 'sagemaker:ListTags')
+
+    source_mapping = {
+        'describe': DescribeJob
```

Comment:
no need for a mapping if its only a describe

Reply:
Good call! It is unlikely that there will ever be config compat for this one. Its not even on the CFN docs. Improved in 4a763fa.